rails/spec/models/theme_spec.rb

```diff
@@ -2,4 +2,11 @@
 
 RSpec.describe Theme, type: :model do
   pending "add some examples to (or delete) #{__FILE__}"
+
+  let(:theme) {Theme.new()}
+  it "can add sponsor logo" do
+    theme.logos.attach(io: File.open("./spec/fixtures/media/terrastories.png"), filename: 'file.pdf')
+    theme.valid?
```

Comment:
not blocking but curious: why not 
expect(theme).to be_valid
?

Reply:
That makes a lot more sense than my method. I think I initially compared the message to the expected message and I ended up at that. When I picked this up, I wasn't aware of #505 which adds another logo, could I make a new issue to update the model to sponsor_logo or something similar? I'll update this test too.

Reply:
@tauhir that sounds like a good consideration! make it an issue and I'll bring it up with the rest of the team to see if that makes sense, makes sense to me though. Definitely feel free to make a follow up PR.